src/index.js

```diff
@@ -22,314 +23,324 @@ const renderEmpty = () => <View/>;
-                if (this._footerHeight !== footerHeight) {
-                  this._footerComponent.setNativeProps({ style: { height: footerHeight }});
-                  this._footerHeight = footerHeight;
-                }
-              }}>
-        { children }
-      </View>
-    );
-  }
-
-  _renderFooterSpacer({ contentBackgroundColor }) {
-    return (
-      <View ref={ref => this._footerComponent = ref } style={{ backgroundColor: contentBackgroundColor }}/>
-    );
-  }
-
-  _maybeRenderStickyHeader({ parallaxHeaderHeight, stickyHeaderHeight, backgroundColor, renderFixedHeader, renderStickyHeader }) {
-    const { viewWidth, scrollY } = this.state;
-    if (renderStickyHeader || renderFixedHeader) {
-      const p = pivotPoint(parallaxHeaderHeight, stickyHeaderHeight);
-      return (
-        <View style={[styles.stickyHeader, { width: viewWidth, ...(stickyHeaderHeight ? { height: stickyHeaderHeight } : null ) }]}>
-          {
-            renderStickyHeader
-              ? (
-                <Animated.View
-                  style={{
-                  backgroundColor: backgroundColor,
-                  height: stickyHeaderHeight,
-                  opacity: interpolate(scrollY, {
-                    inputRange: [0, p],
-                    outputRange: [0, 1],
-                    extrapolate: 'clamp'
-                  })
-                }}>
-                  <Animated.View
-                    style={{
-                    transform: [{
-                      translateY: interpolate(scrollY, {
-                        inputRange: [0, p],
-                        outputRange: [stickyHeaderHeight, 0],
-                        extrapolate: 'clamp'
-                      })
-                    }]
-                  }}>
-                    { renderStickyHeader() }
-                  </Animated.View>
-                </Animated.View>
-              )
-              : null
-          }
-          { renderFixedHeader && renderFixedHeader() }
-        </View>
-      );
-    } else {
-      return null;
-    }
-  }
+		);
+	}
+
+	_renderFooterSpacer({ contentBackgroundColor }) {
+		return (
+            <View ref={ref => this._footerComponent = ref } style={{ backgroundColor: contentBackgroundColor }}/>
+		);
+	}
+
+	_maybeRenderStickyHeader({ parallaxHeaderHeight, stickyHeaderHeight, backgroundColor, renderFixedHeader, renderStickyHeader, scrollingStickyHeader }) {
+		const { viewWidth, scrollY } = this.state;
+		if (renderStickyHeader || renderFixedHeader) {
+			const p = pivotPoint(parallaxHeaderHeight, stickyHeaderHeight);
+			const stickyHeader = scrollingStickyHeader ? (
+                    <Animated.View
+                        style={{
+							transform: [{
+								translateY: interpolate(scrollY, {
+									inputRange: [0, p],
+									outputRange: [stickyHeaderHeight, 0],
+									extrapolate: 'clamp'
+								})
+							}]
+						}}>
+						{this.renderStickyHeader()}
```

Comment:
Fatal redbox error on this.renderStickyHeader() doesn't exist, when prop scrollingStickyHeader is true.

This PR's post comment on scrollingStickyHeader being default to true is also wrong, default is false. Which renders sticky header with new fade behaviour. (awesome improvement at that 🍻)